hire/charles-cole.md

```diff
@@ -50,6 +52,8 @@ education:
     dates: August 2010 - May 2014
     degree: B.A. Classical Greek
 
+graduateProjectURL: https://github.com/char-cole/bloodborne-ally
```

Comment:
We can just use the first project listed as the featured project, or maybe even all of them should show.

Reply:
For this page I think we only want the final project to show, since the goal is being able to display those from a central page. But yes it can definitely just be the first on their list.

We could also eliminate the course field by making sure either JavaScript or C# is the first skill listed in the array of skills.

Reply:
Yeah let’s eliminate the course field